### PR TITLE
Albärt

### PR DESCRIPTION
**Pull request by Albartsol**
# Validate [Albärt](https://solscan.io/token/EUroSPBddnvGhRGWj56S864XWG6wgJ42yLsHSNyAWBj)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://twitter.com/albart_solana/status/1794626206664433999
- Coingecko/ CMC URL (If available): Still working to get Coingecko and CMC set up for Albärt

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [ ] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] I did not modify unrelated code.
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.
- [x] Please make sure your pull request title has your token name. If it just says "Main", or "Validate", it will automatically be closed. PRs containing broken attestation or solscan links will also be closed.

### CODE REVIEW

**Comment by mroizo76** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

 [x] Mint address: EUroSPBddnvGhRGWj56S864XWG6wgJ42yLsHSNyAWBj
 [x] Symbol: Albärt
 [x] Token Decimals: 9
 [x] Token icon: https://sx4dke57tosdo6nonr3ecrmgtsnzccgdtaeta3ym6wpbejgnx5ra.arweave.net/lfg1E7-bpDd5rmx2QUWGnJuRCMOYCTBvDPWeEiTNv2I
 [x] On-chain metadata matches
 [x] No duplication on Jupiter's strict list

**Comment by John76be** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

 [x] Mint address: EUroSPBddnvGhRGWj56S864XWG6wgJ42yLsHSNyAWBj
 [x] Symbol: Albärt
 [x] Token Decimals: 9
 [x] Token icon: https://sx4dke57tosdo6nonr3ecrmgtsnzccgdtaeta3ym6wpbejgnx5ra.arweave.net/lfg1E7-bpDd5rmx2QUWGnJuRCMOYCTBvDPWeEiTNv2I
 [x] On-chain metadata matches
 [x] No duplication on Jupiter's strict list

**Comment by Sky14111997** [comment]
Hello, I am Sky from SkyTech here, confirming the statement above:

 - [x] Mint address: EUroSPBddnvGhRGWj56S864XWG6wgJ42yLsHSNyAWBj
 - [x] Symbol: Albärt
 - [x] Token Decimals: 9
 - [x] Token icon: https://sx4dke57tosdo6nonr3ecrmgtsnzccgdtaeta3ym6wpbejgnx5ra.arweave.net/lfg1E7-bpDd5rmx2QUWGnJuRCMOYCTBvDPWeEiTNv2I
 - [x] On-chain metadata matches
 - [x] No duplication on Jupiter's strict list

**Comment by Depechemo** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

 [x] Mint address: EUroSPBddnvGhRGWj56S864XWG6wgJ42yLsHSNyAWBj
 [x] Symbol: Albärt
 [x] Token Decimals: 9
 [x] Token icon: https://sx4dke57tosdo6nonr3ecrmgtsnzccgdtaeta3ym6wpbejgnx5ra.arweave.net/lfg1E7-bpDd5rmx2QUWGnJuRCMOYCTBvDPWeEiTNv2I
 [x] On-chain metadata matches
 [x] No duplication on Jupiter's strict list

**Comment by Depechemo** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

 - [x] Mint address: EUroSPBddnvGhRGWj56S864XWG6wgJ42yLsHSNyAWBj
 - [x] Symbol: Albärt
 - [x] Token Decimals: 9
 - [x] Token icon: https://sx4dke57tosdo6nonr3ecrmgtsnzccgdtaeta3ym6wpbejgnx5ra.arweave.net/lfg1E7-bpDd5rmx2QUWGnJuRCMOYCTBvDPWeEiTNv2I
 - [x] On-chain metadata matches
 - [x] No duplication on Jupiter's strict list

**Comment by jalaakso** [comment]
I can hereby confirming the statement above and the token is valid.

[x] Mint address: EUroSPBddnvGhRGWj56S864XWG6wgJ42yLsHSNyAWBj
[x] Symbol: Albärt
[x] Token Decimals: 9
[x] Token icon: https://sx4dke57tosdo6nonr3ecrmgtsnzccgdtaeta3ym6wpbejgnx5ra.arweave.net/lfg1E7-bpDd5rmx2QUWGnJuRCMOYCTBvDPWeEiTNv2I
[x] On-chain metadata matches
[x] No duplication on Jupiter's strict list

**Comment by shahal191** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

 - [x] Mint address: EUroSPBddnvGhRGWj56S864XWG6wgJ42yLsHSNyAWBj
 - [x] Symbol: Albärt
 - [x] Token Decimals: 9
 - [x] Token icon: https://sx4dke57tosdo6nonr3ecrmgtsnzccgdtaeta3ym6wpbejgnx5ra.arweave.net/lfg1E7-bpDd5rmx2QUWGnJuRCMOYCTBvDPWeEiTNv2I
 - [x] On-chain metadata matches
 - [x] No duplication on Jupiter's strict list

**Comment by palkko** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

- [x] Mint address: EUroSPBddnvGhRGWj56S864XWG6wgJ42yLsHSNyAWBj
- [x] Symbol: Albärt
- [x] Token Decimals: 9
- [x] Token icon: https://sx4dke57tosdo6nonr3ecrmgtsnzccgdtaeta3ym6wpbejgnx5ra.arweave.net/lfg1E7-bpDd5rmx2QUWGnJuRCMOYCTBvDPWeEiTNv2I
- [x] On-chain metadata matches
- [x] No duplication on Jupiter's strict list

**Comment by Skycrypto11** [comment]
Skyhere, confirms that the information provided is accurate.

 - [x] Mint address: EUroSPBddnvGhRGWj56S864XWG6wgJ42yLsHSNyAWBj
 - [x] Symbol: Albärt
 - [x] Token Decimals: 9
 - [x] Token icon: https://sx4dke57tosdo6nonr3ecrmgtsnzccgdtaeta3ym6wpbejgnx5ra.arweave.net/lfg1E7-bpDd5rmx2QUWGnJuRCMOYCTBvDPWeEiTNv2I
 - [x] On-chain metadata matches
 - [x] No duplication on Jupiter's strict list

**Review by Helenrock — COMMENTED** [review]
I Helen, can hereby attest that the provided information are accurate, and the token is valid.

 - [x] Mint address: EUroSPBddnvGhRGWj56S864XWG6wgJ42yLsHSNyAWBj
 - [x] Symbol: Albärt
 - [x] Token Decimals: 9
 - [x] Token icon: https://sx4dke57tosdo6nonr3ecrmgtsnzccgdtaeta3ym6wpbejgnx5ra.arweave.net/lfg1E7-bpDd5rmx2QUWGnJuRCMOYCTBvDPWeEiTNv2I
 - [x] On-chain metadata matches
 - [x] No duplication on Jupiter's strict list

**Review by Codegenieeth — COMMENTED** [review]
I can hereby attest that the provided information are accurate, and the token is valid.

 - [x] Mint address: EUroSPBddnvGhRGWj56S864XWG6wgJ42yLsHSNyAWBj
 - [x] Symbol: Albärt
 - [x] Token Decimals: 9
 - [x] Token icon: https://sx4dke57tosdo6nonr3ecrmgtsnzccgdtaeta3ym6wpbejgnx5ra.arweave.net/lfg1E7-bpDd5rmx2QUWGnJuRCMOYCTBvDPWeEiTNv2I
 - [x] On-chain metadata matches
 - [x] No duplication on Jupiter's strict list

File: validated-tokens.csv [hunk 1]
```diff
@@ -1613,3 +1613,4 @@ ZACK MORRIS,ZACK,8vCAUbxejdtaxn6jnX5uaQTyTZLmXALg9u1bvFCAjtx7,6,https://cf-ipfs.
 MusCat,MusCat,6PPskLwejGjiRbsrdTvPdo8Ayf6rERxvSezcPDA53TAP,6,https://gateway.irys.xyz/G5wEXOp8IE0Dr5CCWOpN2XqsplXG3_mhTifkUk1tjOE,true
 mockUp Rock,mockUPT,TSTLPnaYJpcjyKaEteb7S13H3XTrCnjZygzuBabuAsV,9,https://raw.githubusercontent.com/uprockcom/mockUPT/main/logo.png,true
 DogWalter,NELSOL,EBPpUYEGsmVGG291MXoXmwucqw6nR9dEUEfK5LJdeuve,6,https://bafybeifalxlv3sydk3w6ud2qlfs3uzpqjsonhlrp23os6qif3appio7qnq.ipfs.cf-ipfs.com,true
+Albärt,Albärt,EUroSPBddnvGhRGWj56S864XWG6wgJ42yLsHSNyAWBj,9,https://sx4dke57tosdo6nonr3ecrmgtsnzccgdtaeta3ym6wpbejgnx5ra.arweave.net/lfg1E7-bpDd5rmx2QUWGnJuRCMOYCTBvDPWeEiTNv2I,true
```
Comment (Codegenieeth) [inline validated-tokens.csv:1616]: BonkDaJeet,JEETS,JEETkCSo9ZYRn6eRv3PwUyoHMq3JpyBJ1kdPYaihTAr7,9,https://ktxg7eykpmnmy2zve6buvhmwwuqlydhhlq4w5sa27dgrszljwvqq.arweave.net/VO5vkwp7GsxrNSeDSp2WtSC8DOdcOW7IGvjNGWVptWE,true

**Review by Codegenieeth — COMMENTED** [review]
BonkDaJeet,JEETS,JEETkCSo9ZYRn6eRv3PwUyoHMq3JpyBJ1kdPYaihTAr7,9,https://ktxg7eykpmnmy2zve6buvhmwwuqlydhhlq4w5sa27dgrszljwvqq.arweave.net/VO5vkwp7GsxrNSeDSp2WtSC8DOdcOW7IGvjNGWVptWE,true

**Comment by plustokens** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

- [x] Mint address: EUroSPBddnvGhRGWj56S864XWG6wgJ42yLsHSNyAWBj
- [x] Symbol: Albärt
- [x] Token Decimals: 9
- [x] Token icon: https://sx4dke57tosdo6nonr3ecrmgtsnzccgdtaeta3ym6wpbejgnx5ra.arweave.net/lfg1E7-bpDd5rmx2QUWGnJuRCMOYCTBvDPWeEiTNv2I
- [x] On-chain metadata matches
- [x] No duplication on Jupiter's strict list

**Comment by Spargelcrypto** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

- [x] Mint address: EUroSPBddnvGhRGWj56S864XWG6wgJ42yLsHSNyAWBj
- [x] Symbol: Albärt
- [x] Token Decimals: 9
- [x] Token icon: https://sx4dke57tosdo6nonr3ecrmgtsnzccgdtaeta3ym6wpbejgnx5ra.arweave.net/lfg1E7-bpDd5rmx2QUWGnJuRCMOYCTBvDPWeEiTNv2I
- [x] On-chain metadata matches
- [x] No duplication on Jupiter's strict list

**Review by BookOfMeme — COMMENTED** [review]
I am BookOfMeme dev, all good here

**Review by Slerfcoin2 — COMMENTED** [review]
All looking fine. Nothing to change. Slerf2 dev here

**Comment by Busded69** [comment]
Hey, I can hereby attest that the provided information are accurate, and the token is valid.

- [x] Mint address: EUroSPBddnvGhRGWj56S864XWG6wgJ42yLsHSNyAWBj
- [x] Symbol: Albärt
- [x] Token Decimals: 9
- [x] Token icon: https://sx4dke57tosdo6nonr3ecrmgtsnzccgdtaeta3ym6wpbejgnx5ra.arweave.net/lfg1E7-bpDd5rmx2QUWGnJuRCMOYCTBvDPWeEiTNv2I
- [x] On-chain metadata matches
- [x] No duplication on Jupiter's strict list

**Review by Albartsol — COMMENTED** [review]
Done

**Review by Helenrock — COMMENTED** [review]
All fine

File: validated-tokens.csv
Comment (Albartsol) [inline validated-tokens.csv:1]: Done

**Review by Albartsol — COMMENTED** [review]
Thanks

**Review by Albartsol — COMMENTED** [review]
Thanks

**Comment by rysekkk** [comment]
> # Validate [Albärt](https://solscan.io/token/EUroSPBddnvGhRGWj56S864XWG6wgJ42yLsHSNyAWBj)
> ## Attestations (Please provide links):
> * Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://twitter.com/albart_solana/status/1794626206664433999
> * Coingecko/ CMC URL (If available): Still working to get Coingecko and CMC set up for Albärt
> 
> ## Validation (Please check off boxes):
> * [x]  The metadata provided in the PR matches what is on-chain (Mandatory)
> * [x]  Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
> * [ ]  Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)
> 
> ## Acknowledgement (Please check off boxes)
> * [x]  I did not modify unrelated code.
> * [x]  My change matches the format in the file (no spaces between fields).
> * [x]  My token is already live and trading on Jupiter.
> * [x]  !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.
> * [x]  Please make sure your pull request title has your token name. If it just says "Main", or "Validate", it will automatically be closed. PRs containing broken attestation or solscan links will also be closed.

**Comment by rysekkk** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

- [x] Mint address: EUroSPBddnvGhRGWj56S864XWG6wgJ42yLsHSNyAWBj
- [x] Symbol: Albärt
- [x] Token Decimals: 9
- [x] Token icon: https://sx4dke57tosdo6nonr3ecrmgtsnzccgdtaeta3ym6wpbejgnx5ra.arweave.net/lfg1E7-bpDd5rmx2QUWGnJuRCMOYCTBvDPWeEiTNv2I
- [x] On-chain metadata matches
- [x] No duplication on Jupiter's strict list

**Comment by rysekkk** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

- [x] Mint address: EUroSPBddnvGhRGWj56S864XWG6wgJ42yLsHSNyAWBj
- [x] Symbol: Albärt
- [x] Token Decimals: 9
- [x] Token icon: https://sx4dke57tosdo6nonr3ecrmgtsnzccgdtaeta3ym6wpbejgnx5ra.arweave.net/lfg1E7-bpDd5rmx2QUWGnJuRCMOYCTBvDPWeEiTNv2I
- [x] On-chain metadata matches
- [x] No duplication on Jupiter's strict list

**Comment by painname** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

[x] Mint address: EUroSPBddnvGhRGWj56S864XWG6wgJ42yLsHSNyAWBj
[x] Symbol: Albärt
[x] Token Decimals: 9
[x] Token icon: https://sx4dke57tosdo6nonr3ecrmgtsnzccgdtaeta3ym6wpbejgnx5ra.arweave.net/lfg1E7-bpDd5rmx2QUWGnJuRCMOYCTBvDPWeEiTNv2I
[x] On-chain metadata matches
[x] No duplication on Jupiter's strict list

**Review by Jymfletzfur — APPROVED** [review]

**Review by 9yoi — APPROVED** [review]

**Pull request merged**